metricbeat/module/elasticsearch/info/info.go

```diff
@@ -0,0 +1,40 @@
+// Fetches basic version info about elasticsearch hosts from http://elasticsearchhost/
+package info
+
+import (
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/metricbeat/helper"
+	"github.com/elastic/beats/metricbeat/module/elasticsearch"
+)
+
+var Config TestMetricConfig
+
+// Metric object
+var Metric = helper.NewMetric("info", Info{}, elasticsearch.Module)
+
+type Info struct {
+	helper.MetricConfig
+	Config TestMetricConfig
```

Comment:
Not anymore I think. I will recheck all the code, but as we discussed, es module should be remove anyways.